pipeline-runner/R/seurat-3-upload_seurat_to_aws.R

```diff
@@ -96,7 +172,7 @@ add_metadata_to_input <- function(scdata, input) {
 }
 
 # get column names that are consistent with sample groups
-find_group_columns <- function(scdata) {
+find_group_columns <- function(scdata, remove.dups = TRUE) {
```

Comment:
unrelated to the PR, but collapse interfaces nicely with dplyr, fndistinct could be used here as well.